test/DocumentFragment.test.php

```diff
@@ -3,8 +3,14 @@
 
 class DocumentFragmentTest extends \PHPUnit_Framework_TestCase {
 
-public function testQuerySelector() {
-	$document = new HTMLDocument();
+    const DOC_CONTENT_BEFORE_INSERT = "<!doctype html><body>"
+    . "<div><ul><li>outOfScope</li></ul></div>"
+    . "<span id='replaceWithSUT'></span>"
+    . "<div><ul><li>outOfScope</li></ul></div>"
+    . "</body>";
```

Comment:
Currently wherever HTML is used, it is done in the Test\Helper class, what do you think of the usage of that class?